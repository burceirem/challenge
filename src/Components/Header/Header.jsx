import React from 'react';
import {Navbar} from 'react-bootstrap';
import'./Header.css';


const Header = () => {
    return (
      <div className="header"  data-testid="header" >
        <Navbar bg="light" >
          <Navbar.Brand href="/">Homepage</Navbar.Brand>            
        </Navbar>
        </div>
    );

};

export default Header;