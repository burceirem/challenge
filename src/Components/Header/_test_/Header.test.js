import React from 'react';
import { render  } from '@testing-library/react';
import renderer from "react-test-renderer";
import Header from '../Header';

it ( "renders header component correctly", () => {
    const { getByTestId } = render(<Header />);
     expect(getByTestId("header")).toHaveClass("header"); 
});


it("matches snapshot",() => {
    const tree = renderer.create(<Header/>);
    expect(tree).toMatchSnapshot();
   })
   