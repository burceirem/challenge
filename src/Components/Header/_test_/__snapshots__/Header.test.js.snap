// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="header"
  data-testid="header"
>
  <nav
    className="navbar navbar-expand navbar-light bg-light"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      Homepage
    </a>
  </nav>
</div>
`;
